### DISCUSSION TITLE
Do we need to distinguish the java_opts of kyuubi and kyuubi-ctl commands?

### DISCUSSION THREAD

**Discussion by wForget** [apache/kyuubi#1422] [Ideas]
The `kyuubi` and `kyuubi-ctl` commands use the same `KYUUBI_JAVA_OPTS` environment variable. Do we need to distinguish them?

like: `KYUUBI_CTL_OPTS` and `KYUUBI_SERVER_OPTS`

**Comment by yaooqinn** [comment]
SGTM

**Reply by wForget** [reply]
Thanks. I will submit a PR for this.
